allergies/allergies.js

```diff
@@ -0,0 +1,66 @@
+//empty arrays for the user inputs to go into
+var healthRestrictions = [];
+var dietRestrictions = [];
+
+/*click event for all elements with the "healthIcon" class, if clicked it updates the class to 
+ reflect that, changes the image to a purple version as needed, and updates the healthRestrictions arrays*/ 
+$('.healthIcon').on('click', function() {
+    if ($(this).hasClass('notClicked')) {
```

Comment:
This if statement could probably be simplified with just a clicked class that is toggled on and off. Also, look at jQuery's toggleClass method.